### PR TITLE
Add unit tests for Bottom Sheet component

### PR DESCRIPTION
**Pull request by luizbaldi**
Why:
* Unit tests are really important on UI libraries

This change addresses the need by:
* Having a more solid and bug-free code base

### CODE REVIEW

File: src/ui/components/BottomSheet/BottomSheet.tsx [hunk 1]
```diff
@@ -61,9 +61,9 @@ const BottomSheet: React.FC<BottomSheetProps> = ({
   height = 400,
   hasOverlay = false,
   isVisible = false,
-  close,
-  children,
-  style,
+  close = () => {},
+  children = null,
+  style = {},
```
Comment (andandandandrew) [inline src/ui/components/BottomSheet/BottomSheet.tsx:66]: What's the reason behind defining defaults for optional props here? I'm not saying it's wrong, would just like to understand your philosophy here.

**Review by andandandandrew — CHANGES_REQUESTED** [review]
Just one question I would like to better understand before giving the 👍

File: src/ui/components/BottomSheet/BottomSheet.tsx (hunk 1 above)
Comment (luizbaldi) [reply src/ui/components/BottomSheet/BottomSheet.tsx:66]: Sure! The thing is that it's a good practice to have default values when the _prop_ isn't required, so we can prevent unexpected behaviors. Does that make sense for you?

There's also a [eslint rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/require-default-props.md) for that :)

**Review by andandandandrew — APPROVED** [review]
@luizbaldi sorry this took me so long to review again! all good to go, and thank you for the clarification on the default values.

**Pull request merged**